core/src/sumeragi/network_topology.rs

```diff
@@ -199,43 +189,28 @@ impl Builder {
     /// Build and get topology.
     ///
     /// # Errors
-    /// 1. Required field is ommitted.
+    /// 1. Required field is omitted.
     /// 2. Not enough peers to be Byzantine fault tolerant
```

Comment:
it looks to me as if `2.` is not longer possible?

Reply:
why don't we return error anymore when constructing network topology if it's not byzantine fault tolerant?

Reply:
Because now the tolerance is calculated from the topology so the topology is always somewhat BFT